src/disign.css

```diff
@@ -0,0 +1,58 @@
+.List {
+    width:400px;
+    /*height: 200px;*/
+    background: #9ee1e1;
+    float:left;
+    margin: 0 15px 15px 0;
+    text-align:center;
+    padding: 10px;
+    list-style-type: none;
+}
+
+.UserInfoLeft {
+    background: #50e3e3;
+    border-radius: 15px;
+    /*border: 2px solid #73AD21;*/
+    text-align: left;
+    padding-left: 15px;
+    padding-right: 15px;
+    margin-bottom: 1em; /* Расстояние снизу */
+}
+
+.UserInfoRight {
+    border-radius: 15px;
+    /*border: 2px solid #73AD21;*/
+    text-align: left;
+    padding-left: 15px;
+    padding-right: 15px;
+    margin-bottom: 1em; /* Расстояние снизу */
```

Comment:
Using 'em' and 'px' at the same time looks weird.
Please choose one of them, 
Do you understand the difference between "em"/"px"?
"em" is a relative size, while "px" is an absolute size.
https://www.w3schools.com/cssref/css_units.php